### ISSUE TITLE
Blocking control exception due to page longer than browser window

### ISSUE THREAD

**Issue by ghost** [icnocop/cuite#101]
I have looked at codeples, the interweb and git issues, but could not find an answer.

We have a number of web pages longer than the browser window.  When using CodedUI, we would have to press the page down button to expose the controls, or else we would get an exception with "Another control is blocking the control".   A brittle solution.

We are wanting to use the page object pattern and putting PageDown sendkeys in the object not is only brittle, but ties the page to the browser.

What is the suggested method for exposing HTML controls for web pages that are longer/bigger than the browser window?

thanks

**Comment by FantasticFiasco** [comment]
@icnocop Have you experienced this issue before?

@TheBigAvanger  It would benefit me greatly if you where able to create a sample application with a test that fails due the blocking error.

**Comment by ghost** [comment]
Yes we have this issue with Coded UI as well.  We have to use page down or ctl-end to move to the end of the form.

 Easy to recreate.  Make a web form page 2-3 times longer than the browser window.  Have a unit test load up the window.  Enter data in the top most control, then the bottom most control (on the form).  You will get the exception.

**Comment by JSDreyer** [comment]
I have run into this issue with Coded UI.  Usually when you're automating a long page in IE, Coded UI will jump to the next object, scrolling down just enough to reveal the object at the very bottom of the browser display window.  So usually it works fine.  However, when there's a toolbar dialog at the bottom of the screen (i.e. Save this password?, Only secure content is displayed, Disable add-ons, etc.), that dialog will cover the control triggering the error.  

My solution has been to go into the browser settings and disable all of these popups.  It's easier than writing WinForm automation to handle them.

Another possibility might be an internal scrollbar in a control (say an embedded list or combo box) that prevents the automation from scrolling down.  I haven't run into this in Coded UI, but have in other automation tools.

**Comment by FantasticFiasco** [comment]
Thanks @JSDreyer, these are all goods tips!
